quickevent/app/plugins/Runs/src/runsplugin.h

```diff
@@ -84,11 +84,11 @@ class RUNSPLUGIN_DECL_EXPORT RunsPlugin : public qf::qmlwidgets::framework::Plug
 	Q_INVOKABLE bool exportResultsCsosStage(int stage_id, const QString &file_name);
 	Q_INVOKABLE bool exportResultsCsosOverall(int stage_count, const QString &file_name);
 
-	qf::core::utils::TreeTable startListClassesTable(const QString &where_expr, bool insert_vacants);
-	qf::core::utils::TreeTable startListClubsTable();
+	qf::core::utils::TreeTable startListClassesTable(const QString &where_expr, const bool insert_vacants, const bool day_time_start);
```

Comment:
`qf::core::utils::TreeTable startListClassesTable(const QString &where_expr, const bool insert_vacants, const StartTimeFormat start_time_format);`

Reply:
StartTimeFormat je definovanej v ramci jedne tridy. To ho jako fakt chces zatahnout do celyho QE ?

Reply:
presne tak, protoze volani funkce `startListClassesTable("stNum>0", true, false)` neni moc samovysvetlujici a pokud uz jsi enum zavedl, proc ho nepouzit? Obzvast, kdyz se z nazvu parametru `day_time_start` jen tezko uhodne, co by to mohlo znamenat. Uznavam, se je to trochu opruz, ale je potreba myslet ne ty, co to budou udrzovat pripadne po nas, at to maji co nejcitelnejsi.

Reply:
Dobre. Prekopu to na pouziti enumu primo.